app/src/main/res/values-ca/strings.xml

```diff
@@ -690,55 +690,55 @@
     <string name="alarmMode_alarm">Alarma</string>                <!-- alarm type (setting) -->  
     <string name="alarmMode_notification">Notificació</string>                <!-- alarm type (setting) -->
 
-    <string name="configLabel_alarmEdit_scheduleTray">Schedule:</string>               <!-- group title -->  <!-- TODO -->
-    <string name="configLabel_alarmEdit_onAlertTray">On Alert:</string>               <!-- group title -->  <!-- TODO -->
-    <string name="configLabel_alarmEdit_onDismissTray">On Dismiss:</string>               <!-- group title -->  <!-- TODO -->
+    <string name="configLabel_alarmEdit_scheduleTray">Planificació:</string>               <!-- group title -->  
+    <string name="configLabel_alarmEdit_onAlertTray">Al activar-se:</string>               <!-- group title -->  
+    <string name="configLabel_alarmEdit_onDismissTray">Al rebutjar:</string>               <!-- group title -->  
 
     <string name="alarmOption_vibrate">Vibrar</string>                <!-- checkbox text -->
     <string name="alarmOption_ringtone_none">Sense so</string>          <!-- ringtone label (none) -->  
-    <string name="alarmOption_ringtone_label"><xliff:g id="icon">[icon]</xliff:g> <xliff:g id="name">%1$s</xliff:g></string>  <!-- ringtone label format -->
+    <string name="alarmOption_ringtone_label"><xliff:g id="icon">[icona]</xliff:g> <xliff:g id="name">%1$s</xliff:g></string>  <!-- ringtone label format -->
```

Comment:
Try to avoid changing text between the `<xliff:g>` tags. In this case the `[icon]` tag is replaced by an actual icon just before its displayed.

I think that is the only note I have. This looks like good work, thank you.

Reply:
Sorry, I missed that. The rest remains untouched.